roles/infrastructure/defaults/main.yml

```diff
@@ -70,3 +72,8 @@ infrastructure_managed_disk_type: Premium_LRS
 infrastructure_os_disk_size_gb: 100
 
 infrastructure_aap_installer_inventory_path: "/opt/ansible-automation-platform/installer/inventory_azure"
+
+# Load balancer
+infrastructure_create_controller_lb: false
+infrastructure_cert_path: ""
+infrastructure_cert: "{{ lookup('file', lookup('env', 'HOME') + infrastructure_cert_path) | b64encode }}"
```

Comment:
```suggestion
infrastructure_cert: "{{ lookup('file', infrastructure_cert_path) | b64encode }}"
```
I would just make the path be a full path since there's not a known / common place to look.